tests/test_add_new_openwrap_partner.py

```diff
@@ -1209,7 +1209,7 @@ def test_setup_partner_for_RON(self, mock_get_users, mock_get_root_ad_unit_id,
                                                setup_type = constant.NATIVE, creative_template = None, num_creatives=2,
                                                use_1x1=False, currency_code='USD', custom_targeting= None,
                                                same_adv_exception= False, device_categories=None, device_capabilities=None, 
-                                               roadblock_type= 'ONE_OR_MORE', slot = None,adpod_creative_durations=None, creative_user_def_var = None, deal_config=None, deal_lineitem_enabled= False,  deal_config_type = None)
+                                               roadblock_type= 'ONE_OR_MORE', slot = None,adpod_creative_durations=None, creative_user_def_var = None, deal_config=None, deal_lineitem_enabled= False,  deal_config_type = None, video_position_type=None)
```

Comment:
Also add test case for dfp.create_line_item_config

Reply:
Done